### PR TITLE
210 Testing user profile

### PR DESCRIPTION
**Pull request by Fava00**
Testing user profile:
    Validation with edge cases.
    Tests for update, delete, and user profile getting.

### CODE REVIEW

File: backend/src/test/java/com/tcs/dhv/userProfileTest/UserProfileEndToEndTest.java [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class UserProfileEndToEndTest {
+
+    @Autowired
```
[the remark this answers is not in the document]
Comment (Fava00) [reply backend/src/test/java/com/tcs/dhv/userProfileTest/UserProfileEndToEndTest.java:38]: I mean I can remove it, I dont need to refactor it much

**Review by IvanIsaev-12 — APPROVED** [review]
